public/app/features/alerting/unified/mocks/server/entities/alertmanager-config/grafana-alertmanager-config.ts

```diff
@@ -124,6 +124,8 @@ const grafanaAlertmanagerConfig: AlertManagerCortexConfig = {
               addresses:
                 'gilles.demey+1@grafana.com, gilles.demey+2@grafana.com, gilles.demey+3@grafana.com, gilles.demey+4@grafana.com',
               singleEmail: false,
+              message: '{{ template "slack-template" . }}',
+              subject: 'some custom value',
```

Comment:
Means we get this when using the mock API:
![image](https://github.com/user-attachments/assets/740dc4c7-d0a5-4bf7-b315-2f8b69d0dc8c)